package.json

```diff
@@ -107,15 +107,14 @@
   },
   "license": "GPL-3.0-or-later",
   "resolutions": {
-    "@celo/wallet-walletconnect": "1.2.0",
```

Comment:
I wasn't sure why this was here, but i was worried it could cause issues with react-celo (because it's such an old version), so I removed it.